src/content/app/tools/vep/types/vepSubmission.ts

```diff
@@ -18,25 +18,55 @@ import type { CommittedItem } from 'src/content/app/species-selector/types/commi
 
 export type VepSelectedSpecies = Omit<CommittedItem, 'isEnabled'>;
 
+type ClientSideSubmissionStatus =
+  | 'NOT_SUBMITTED' // initial status of a submission while it is being prepared by the user
+  | 'SUBMITTING' // during the time between user clicking the submit button, and the submission being accepted by the backend
+  | 'FAILED_ON_SUBMISSION' // some unspecified (most likely network timeout) error happened during submission
```

Comment:
How about `SUBMISSION_FAILED` instead?

Reply:
Couldn't this be misinterpreted as a problem for the submission to successfully run to completion (what is currently represented by the `FAILED` server-side submission status). I wanted to express that here the tools api did not accept the submission for some reason (a network failure, etc.). `SUBMISSION_REJECTED`?

Reply:
`UNSUCCESSFUL_SUBMISSION`, or `SUBMISSION_UNSUCCESSFUL` was another suggested option, which can work.